skt/reporter.py

```diff
@@ -293,21 +293,23 @@ def __getjobresults(self):
                     failed_tasks.append('/distribution/kpkginstall')
                 else:
                     recipe_tests = runner.get_recipe_test_list(recipe)
-                    result += ['We ran the following tests:']
-                    for test_name in recipe_tests:
-                        task_node = recipe.find(
-                            "task[@name='{}']".format(test_name)
-                        )
-                        test_result = task_node.attrib.get('result')
-                        result += ['  - {}: {}'.format(test_name,
-                                                       test_result.upper())]
-                        if test_result != 'Pass':
-                            failed_tasks.append(test_name)
-
-                        if task_node.find('fetch') is not None:
-                            result.append('    - Test URL: {}'.format(
-                                task_node.find('fetch').attrib.get('url')
-                            ))
+                    if recipe_tests:
```

Comment:
Is adding this condition necessary for the main change? Or is this just fixing something related?

Reply:
There's a possibility all the tests that were supposed to be ran had to be skipped, so there are no actual tests that were run. The condition makes sure we don't say "We ran the following tests" followed by an empty line in the report.

Reply:
Makes sense, thank you, Veronika :)